src/util.c

```diff
@@ -297,6 +297,19 @@ void do_sleep(unsigned int sec)
 		;
 }
 
+void do_usleep(unsigned int usec)
+{
+	struct timespec deadline;
+	clock_gettime(CLOCK_MONOTONIC, &deadline);
```

Comment:
Nitpick, should be an empty line after declarations.

Reply:
Will change that in V2!